Livecheckables/libcuefile.rb

```diff
@@ -0,0 +1,6 @@
+class Libcuefile
+  livecheck do
+    url "https://www.musepack.net/index.php?pg=src"
+    regex(/href=.*?libcuefile[._-]v?(r\d+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?libcuefile[._-](r\d+)\.t/i)
```

Based on the file name format, I don't imagine we'll encounter a file name like `libcuefile_vr123.tar.gz`. I'm fine with omitting the`v?` here, as it doesn't use a common version scheme.